portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,17 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Kwame Ampadu-Boateng's Portfolio</h1>
+      <p>Hello! My name is Kwame Ampadu-Boateng. I a currently a rising 
+          sophomore at Vanderbilt University, majoring in Computer Science and 
+          Economics. This is a website I made throught Google's STEP(Student 
+          Training in Engineering Program).</p>
+        <img src="/images/Self.jpg" width="300" height="350" 
```

Comment:
Indent the `<img>` tag back one level. All children within a given element should have the same indentation.